frontend/src/modules/signup/view.jsx

```diff
@@ -189,11 +189,12 @@ const SignUp = ({ match: { params }, ...props }) => {
     meaOptions,
     formStep,
     formEdit,
+    profile,
   } = storeData;
 
   const formData = signUpData.useState();
   const { editId, data } = formData;
-  const { status, id } = formEdit.initiative;
+  const { status, id } = formEdit.signupStep;
```

Comment:
Found error here `TypeError: Cannot destructure property 'status' of 'formEdit.signupStep' as it is undefined.`
And I found on `store.js` the flag is `signUp` not `signupStep`.

Reply:
great catch! thanks!
PS: added one commit